src/node/miner.cpp

```diff
@@ -252,7 +252,7 @@ void BlockAssembler::addChunks()
 
     while (selected_transactions.size() > 0) {
         // Check to see if min fee rate is still respected.
-        if (chunk_feerate.fee < m_options.blockMinFeeRate.GetFee(chunk_feerate_vsize.size)) {
+        if (chunk_feerate_vsize < m_options.blockMinFeeRate.GetFeePerVSize()) {
```

Comment:
ff2a16af040ac0906b4ebd488b1125b6ea25fa14

I don't think we should be tie-breaking based on size here?

```Suggestion
        if (chunk_feerate_vsize << m_options.blockMinFeeRate.GetFeePerVSize()) {
```